<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>screen_writingClass</class>
 <widget class="QMainWindow" name="screen_writingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>screen_writing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_Name">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_descr">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_act">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Name_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Descr</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Name_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Age</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>290</y>
      <width>75</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>341</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>290</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>280</y>
      <width>75</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Update
Ideas</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>526</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="screen_writing.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>screen_writingClass</receiver>
   <slot>addButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>screen_writingClass</receiver>
   <slot>deleteButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>screen_writingClass</receiver>
   <slot>updateButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addButton()</slot>
  <slot>deleteButton()</slot>
  <slot>updateButton()</slot>
 </slots>
</ui>
